app/views/users/index.html.erb

```diff
@@ -1,2 +1,18 @@
-<h1>All Users</h1>
-<!-- Placeholder for listing all users -->
+<h2>All Users<h2>
+<main id="home" class="home-page">
+  <ul class="users-list">
+    <% User.all.each do |user| %>
+      <li class="user-container">
+        <div class="profile-pic">
+          <%= link_to user_path(user) do %>
+            <img src="<%= user.photo %>" alt="profile-picture">
+          <% end %>
+        </div>
+        <div class="class-info">
+          <h3><%= link_to user.name, user_path(user)  %></h3>
+          <p class="posts-counter">Number of posts: <%= user.posts_counter || 0 %></p>
+        </div>
+      </li>
+    <% end %>
+  </ul>
+</main>
```

Comment:
| @WilhelmK109 User Index Page | Microverse User Index Page |
|-|-|
| ![image](https://github.com/WilhelmK109/BlogApp/assets/93039359/5a8854dc-0cbf-45c7-9955-73a2a9945c6d) | ![image](https://github.com/WilhelmK109/BlogApp/assets/93039359/9c7a522b-3256-48c8-ad1e-9fbfa105044d) |

- [ ] Great work so far but here we need minor changes 😊 Kindly expand your data on the whole screen as you can see in the given design 👀 that the Name is on the left side and Posts Counter is on the right side 🤔 to organize the user information well 👍 Kindly make this change 📝